###############---------------------------------------------国内版-----------------------------------------------------#################
asr:                                    # ASR node parameters / 语音节点参数
  ros__parameters:
    VAD_MODE: 2                         # VAD sensitivity / vad灵敏度
    sample_rate: 16000                  # ASR recording audio sample rate / asr录音音频采样率
    frame_duration_ms:  30              # VAD frame size in ms / vad帧大小单位ms
    use_oline_asr: False                # Whether to use online ASR (True for online, False for offline) / 是否使用在线asr识别（True使用在线，False使用离线）
    mic_serial_port: "/dev/ttyUSB0"     # Microphone serial port alias / 麦克风串口别名
    mic_index: 0                        # Microphone index / 麦克风索引
    language: 'zh'                      # ASR language / asr语言
    regional_setting : "China"          #international：国际版  China：国内版 # international: International version, China: Domestic version

model_service:                          # Model service node parameters / 模型服务器节点参数
  ros__parameters:
    language: 'zh'                      # Large model interface language / 大模型接口语言
    useolinetts: False                   # Whether to use online TTS (True for online, False for offline) / 是否使用在线语音合成（True使用在线，False使用离线）
    llm_platform: 'ollama'              # Currently selected large model platform / 当前选用的大模型平台
    # Available platforms: 'ollama', 'tongyi', 'spark', 'qianfan' / 可选平台: 'ollama', 'tongyi', 'spark', 'qianfan'
    regional_setting : "China"          #international：国际版  China：国内版 # international: International version, China: Domestic version
    camera_width: 640                   # Camera frame width / 摄像头画面宽度
    camera_height: 480                  # Camera frame height / 摄像头画面高度
    camera_type: 'usb'                 # Camera type: 'csi', 'usb' / 摄像头类型: 'csi', 'usb'

tts_only_node:                          # TTS only node parameters / 纯语音合成节点参数
  ros__parameters:
    language: 'zh'                      # TTS language / 语音合成语言
    useolinetts: False                   # Whether to use online TTS / 是否使用在线语音合成
    regional_setting : "China"          #international：国际版  China：国内版 # international: International version, China: Domestic version


###############---------------------------------------------International Edition-----------------------------------------------------#################
# asr:                                    # ASR node parameters / 语音节点参数
#   ros__parameters:
#     VAD_MODE: 2                         # VAD sensitivity / vad灵敏度
#     sample_rate: 16000                  # ASR recording audio sample rate / asr录音音频采样率
#     frame_duration_ms:  30              # VAD frame size in ms / vad帧大小单位ms
#     use_oline_asr: False                # Whether to use online ASR (True for online, False for offline) / 是否使用在线asr识别（True使用在线，False使用离线）
#     mic_serial_port: "/dev/ttyUSB0"     # Microphone serial port alias / 麦克风串口别名
#     mic_index: 0                        # Microphone index / 麦克风索引
#     language: 'en'                      # ASR language / asr语言
#     regional_setting : "international"          #international：国际版  China：国内版 # international: International version, China: Domestic version

# model_service:                          # Model service node parameters / 模型服务器节点参数
#   ros__parameters:
#     language: 'en'                      # Large model interface language / 大模型接口语言
#     useolinetts: False                   # Whether to use online TTS (True for online, False for offline) / 是否使用在线语音合成（True使用在线，False使用离线）
#     llm_platform: 'openrouter'              # Currently selected large model platform / 当前选用的大模型平台
#     # Available platforms: 'ollama', 'openrouter' / 可选平台: 'ollama',  'openrouter'
#     regional_setting : "international"          #international：国际版  China：国内版 # international: International version, China: Domestic version
#     camera_width: 640                   # Camera frame width / 摄像头画面宽度
#     camera_height: 480                  # Camera frame height / 摄像头画面高度
#     camera_type: 'usb'                 # Camera type: 'csi', 'usb' / 摄像头类型: 'csi', 'usb'

# tts_only_node:                          # TTS only node parameters / 纯语音合成节点参数
#   ros__parameters:
#     language: 'en'                      # TTS language / 语音合成语言
#     useolinetts: False                   # Whether to use online TTS / 是否使用在线语音合成
#     regional_setting : "international"          #international：国际版  China：国内版 # international: International version, China: Domestic version
